posts/2020/03/Spyder-4.md

```diff
@@ -0,0 +1,31 @@
+
+Besides, when the future looked grim, after my team and I were let go from Anaconda at the end of 2017 (not by Travis decision), a lot of users came to our rescue by making donations through our [Open Collective page](https://opencollective.com/spyder). That, and a Numfocus development grant we received next year, filled us with confidence and allowed us to continue with Spyder's development in 2018, even after losing part of our team in the process.
+
+Last year it was also a pleasant surprise to learn that several companies were interested in seeing Spyder prosper and thrive. Through Quansight Labs and its [Community Working Order](https://labs.quansight.org/blog/2019/05/community-driven-opensource-funded-development/) concept, we were able to sign contracts with two of them: [TDK-Micronas](https://www.micronas.tdk.com/en) and [Kite](https://kite.com/). Their support was critical to finish Spyder 4 because it allowed me to hire most of my old Anaconda team back, plus two new additions, to work full-time on the project. Therefore, I can't them enough for showing up just at the right time!
+
+And finally, even when I am the public face of Spyder, due to my presence in our [issues tracker](https://github.com/spyder-ide/spyder/issues) and [StackOverflow](https://stackoverflow.com/users/438386/carlos-cordoba), it's really the Spyder team that is in charge of implementing new features and fixing most bugs. So my last round of acknowledgments goes to them. I was fortunate enough to convince some of the most talented Colombian software developers to work for the project, and they did a hell of a job in this release!
+
+For instance, [Gonzalo Peña-Castellanos](https://github.com/goanpeca/) helped us to greatly improve the usability experience of code completion and linting in the editor, to implement most enhancements to the Files pane, and to refactor and put in good shape our configuration system; [Edgar Margffoy](https://github.com/andfoy) single-handedly created a client to support the same [protocol](https://microsoft.github.io/language-server-protocol/) used by VSCode to provide completion and linting for lots of programming languages, added snippet completions and vastly improved code folding in the editor; [Daniel Althviz](https://github.com/dalthviz) developed the necessary infrastructure to install and use Kite smoothly within Spyder, and added the new object explorer viewer to the Variable Explorer; and our Junior developers, [Stephannie Jimenez](https://github.com/steff456) and [Juanita Gomez](https://github.com/juanis2112), although still learning their way in our complex source code, managed to do important contributions, such as improving the icons we use per file type in Files (Juanita), and allowing to run code in an external system terminal on macOS (Stephannie).
+
+I hope you enjoy the results of this massive effort! And happy Spydering!!
```

Comment:
```suggestion
I hope you all enjoy the results of this massive effort! And happy Spydering!!!
```

Pro tip: In English its convention to either use one exclamation point, or (very sparingly) three. Two just looks like a mistake.

Reply:
Thanks for the suggestion!